#1/ClixpesaMint.md

```diff
@@ -0,0 +1,27 @@
+# Clixpesa Mint
+
+## Former Participation in Celo Hackathons
+
+- Link to hackathon submission
+
+## Goal for this Proof of Ship
+### Clixpesa smart contracts redesign, upgrades and overdraft feature.
+
```

Comment:
This is a great goal. I just want to remind you, that the project needs to be open source.